src/eq_fault_geom/geomio/cfm_shp_to_vtk.py

```diff
@@ -0,0 +1,214 @@
+from pathlib2 import Path
```

Comment:
This adds a new requirement which isn't really needed?? this can be done using os or pathlib which are in core python.

FYI I substituted pathlib2 with pathlib and it worked fine for me.

Reply:
Yes, I think I initially put that in for backward compatibility with Python 2, but I guess we won't be worrying about that.